docs/source/configuration.rst

```diff
@@ -42,21 +42,38 @@ on both TI1 and TI2 edges.
 Pinout
 ======
 
-====  =========  ==============================
-Pin   Function   Connected to
-====  =========  ==============================
-PA8   TIM1_CH1   Left motor encoder, channel A
-PA9   TIM1_CH2   Left motor encoder, channel B
-PB0   TIM3_CH3   Left motor PWM
-PB1   TIM3_CH4   Right motor PWM
-PB6   TIM4_CH1   Right motor encoder, channel B
-PB7   TIM4_CH2   Right motor encoder, channel A
-PB10  USART3_TX  Bluetooth RX
-PB11  USART3_RX  Bluetooth TX
-PB12  GPIO       Left motor driver IN1
-PB13  GPIO       Left motor driver IN2
-PB14  GPIO       Right motor driver IN2
-PB15  GPIO       Right motor driver IN1
+====  =========  ======  ==============================
+Pin   Function   Type	 Connected to
+====  =========  ======  ==============================
+PA0   ADC12_IN0  Input   Battery 1
+PA1   ADC12_IN1	 Input	 Battery 2
+PA2   ADC12_IN2	 Input 	 Gyroscope Vref	
```

Comment:
I'd use spaces here instead of tabs (for consistency). :wink:

Reply:
Right.